common/app/controllers/IndexControllerCommon.scala

```diff
@@ -6,6 +6,7 @@ import play.api.mvc._
 import services.Index
 import services.IndexPage
 import performance.MemcachedAction
+import views.support.RenderOtherStatus
```

Comment:
good point, it only deals with errors so should probably be named as such.
there's a bit more to do on this, I'll look at renaming it then.